web_src/less/_base.less

```diff
@@ -895,7 +895,7 @@ footer {
     .ui.menu.new-menu {
         overflow-x: auto !important;
         justify-content: left !important;
-        padding-bottom: 5px;
+        padding-bottom: 2px;
```

Comment:
What does this change?

Reply:
On small screen there's a overflow mechanism. Previous:

<img width="129" alt="image" src="https://user-images.githubusercontent.com/115237/85235637-f3c0cb00-b416-11ea-9faf-10c25155d906.png">

After (just fit the border):

<img width="130" alt="image" src="https://user-images.githubusercontent.com/115237/85235639-fc190600-b416-11ea-9ae7-d898d0652334.png">